src/functions/template-tags/date.php

```diff
@@ -342,6 +349,13 @@ function tribe_get_end_date( $event = null, $display_time = true, $date_format =
 			return '';
 		}
 
+		static $end_dates = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 14515b96a14917b4e9343209232bb7f7</sub>